index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap');
+      </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main" class="container">
```

Comment:
I think it would be better if you use a `section` tag instead of a `div` tag 🚀 because it is important to make your HTML page semantic and follow the best practice 🚀✅✅🔥🔥